types/Cargo.toml

```diff
@@ -21,7 +21,7 @@ ed25519-dalek = { version = "1.0.0", default-features = false, features = ["rand
 failure = { version = "0.1.8", default-features = false, features = ["failure_derive"] }
 hex = { version = "0.4.2", default-features = false }
 hex_fmt = "0.3.0"
-k256 = { version = "0.4.2", default-features = false, features = ["ecdsa", "zeroize"] }
+libsecp256k1 = "0.3.5"
```

Comment:
This has the feature `std` enabled by default - I think we'll need to change that so that we have `default_features = false` here and enable `std` for this when our `std` feature is enabled.

Reply:
Fixed in 8c3d61879143a7ef88bb839bdcf8a6ebdaaba02d